### DISCUSSION TITLE
Default colorscheme has little markdown syntax highlighting

### DISCUSSION THREAD

**Discussion by avidseeker** [neovim/neovim#29995] [Ideas]
`$ nvim --clean +'colorscheme default test.md'
![image](https://github.com/user-attachments/assets/87eabb4e-47c4-42a3-80b9-53138c195ba9)

Compare with `:colorscheme vim`
![image](https://github.com/user-attachments/assets/587d0a81-8d72-4dec-bd94-e525e53cfd99)

or desert:
![image](https://github.com/user-attachments/assets/9521f2bd-5296-4b08-96df-1e29eb8a2d95)

**Comment by clason** [comment]
What's your question? The colorscheme is intentionally minimal; this is hard design choice and will not be revisited.

**Comment by avidseeker** [comment]
Since Neovim colorscheme is relatively new, with the goal (I suppose?) of having sane defaults for new users and non-riced initrc, I thought I'd share my feedback on the colorscheme.

> this is hard design choice and will not be revisited.

If that's the case, I'll mark this as resolved.

**Reply by clason** [reply]
Yes, sorry, that ship has sailed. There was a _long_ discussion pre-0.10, and the primary goal had become branding. (It was never meant to be default in the sense of "replace the need for custom colorschemes"; more of "no more screaming pink when reproducing something without a config.)
